scripts/src/repl-client.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
```

Comment:
@victordoc @shaharsol (and @doodyparizada why not).
Funny anecdote, I had an internet outage today. Finally had time now to see what the problem was, entered my router's web interface, which showed that the internet connection is up and running. Yet nothing loads. I try to ping 8.8.8.8 (and 1.1.1.1) from my computer but hit an ICMP request timeout (so there's a connection but something is wrong). Since the the router says it's connected I wanted to try and ping one of the above IPs from it. The problem, MacOS does not come with telnet preinstalled and I don't have an internet connection to install it. I desperately need a telnet like TCP client, AND that's when I remembered this simple client I wrote a week or so ago. Which worked magnificently! exactly how I would expect the Telnet client to (I even installed and compared them, the only difference was a few characters this client prints out when the connection is initially made). So, in the future you can use this client with port 23, and it should do the trick. That's it, you don't have to install telnet now. Of course, you can also use Telnet instead of this client to connect to the remote REPL (just specify the correct port)